pontoon/base/models.py

```diff
@@ -267,7 +267,13 @@ def get_latest_activity(self, project=None):
             else:
                 return None
         else:
-            return ProjectLocale.get_latest_activity(project, self)
+            if hasattr(self, 'fetched_project_locales'):
```

Comment:
I agree. And I also don't know what's the best practice around this. :) I'm trying to make custom attribute names start with `fetched_` so they are at least easier to spot.